src/clj/runbld/tests/junit.clj

```diff
@@ -88,9 +88,10 @@
 
 (defn find-failures [dir filename-format]
   ;; find the xml files
-  (debug/log "Searching for junit test output files")
+  (rio/log "Searching for junit test output files with the pattern:"
```

Comment:
https://www.cbsnews.com/news/blue-spixs-macaw-parrot-that-inspired-rio-is-extinct-in-wild/